flutter/lib/src/binding_wrapper.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:developer';
+
+import 'package:flutter/widgets.dart';
+
+import '../sentry_flutter.dart';
+
+// The methods and properties are modelled after the the real binding class.
+class BindingWrapper {
+  /// The current [WidgetsBinding], if one has been created.
+  /// Provides access to the features exposed by this mixin.
+  /// The binding must be initialized before using this getter;
+  /// this is typically done by calling [runApp] or [WidgetsFlutterBinding.ensureInitialized].
+  WidgetsBinding? get instance {
+    try {
+      return _ambiguate(WidgetsBinding.instance);
+    } catch (e, s) {
+      log(
+        'WidgetsBinding.instance was not yet initialized',
+        level: SentryLevel.error.toDartLogLevel(),
+        error: e,
+        stackTrace: s,
+        name: 'Sentry',
+      );
```

Comment:
What if `BindingWrapper` takes a `HubAdapter` so we can `hub.options.logger(...)`.
Otherwise, we just remove this call, the SDK should use its own logger.